server/events.go

```diff
@@ -832,14 +832,57 @@ func (s *Server) leafNodeConnected(sub *subscription, _ *client, subject, reply
 	}
 }
 
+// Common filter options for system requests STATSZ VARZ SUBSZ CONNZ ROUTEZ GATEWAYZ LEAFZ
+type ZFilterOptions struct {
+	Name    string `json:"server"`
```

Comment:
`name` for the key might be better since that's what it is, we have ambiguity in what "server" means, so being specific might help

Reply:
Can we also add `host` here?

Reply:
incorporated name change and added host